validphys2/src/validphys/config.py

```diff
@@ -216,6 +218,10 @@ def parse_use_cuts(self, use_cuts: (bool, str)):
 
         return res
 
+    def produce_replicas(self, nreplica: int):
```

Comment:
They are not equal though, the n3fit generates from a to b while this one is 1 to N.

Reply:
Ah yes of course, but won't this cause problems if they're called the same thing?

Reply:
E.g I believe some functions in `n3fit_data.py` collect over the "n3fit" `pseudoreplicas` e.g

https://github.com/NNPDF/nnpdf/blob/1b9dd82a3e9a7e7a1891d7c43ee82bf0af909856/validphys2/src/validphys/n3fit_data.py#L474

Reply:
They are replicas just the same only the production of them changes. Maybe adding a default initial = 1 to `produce_replicas` would work (so that they can be made truly the same)

Reply:
What does reportengine think @Zaharid

Reply:
What does reportengine think @Zaharid

Reply:
I believe reportengine would ignore one of the two `replicas` specifications as it is now. I suppose n3fit could override it with a custom namespace, but looks a bit dicey.

Reply:
It is what `n3fit` is doing. Anyway, that would be for another PR but the solution is for `n3fit` to set the final replica and the initial replica instead of creating its own range.